src/molecules/formfields/CheckBoxField/index.js

```diff
@@ -27,14 +27,15 @@ class CheckBoxField extends React.Component {
           checked={input.value}
           {...input}
         />
-        <Text
-          tag='span'
-          bold={input.checked}
-          color={input.checked ? 'primary-3' : textColor || 'neutral-2'}
-          className={styles['checkbox-label']}
-        >
-          { label }
-        </Text>
+
+        <span className={styles['checkbox-label']}>
```

Comment:
I would not change this please, I think it will break feature specs. The `tag` prop on `Text` should render a `span` for you. Double check, because it's working for me